Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Image_res_dialog.ui

```diff
@@ -43,27 +31,76 @@
     </widget>
    </item>
    <item row="0" column="1">
-    <widget class="QComboBox" name="precisionList">
-     <property name="editable">
-      <bool>false</bool>
+    <widget class="QComboBox" name="imageType"/>
+   </item>
+   <item row="0" column="0">
+    <widget class="QLabel" name="label_3">
+     <property name="text">
+      <string>Please choose the image type</string>
      </property>
     </widget>
    </item>
-   <item row="1" column="0">
-    <widget class="QLabel" name="label_2">
-     <property name="font">
-      <font>
-       <pointsize>11</pointsize>
-       <italic>true</italic>
-      </font>
+   <item row="2" column="0" colspan="2">
+    <widget class="QGroupBox" name="groupBox">
+     <property name="title">
+      <string>GroupBox</string>
```

Comment:
I want to comment on the modification of this dialog in this PR.

I wonder why there is a group box, in this dialog. Group boxes are used to group widgets, and separate them from other widgets in UI. But in this dialog there is almost nothing to separate from! That widget can use just a `QWidget`, that is shown or not depending on the content of the `QComboBox imageType`.

And the group box title should not have been let to the default "GroupBox".

... Anyway, do not modify that. I am working on another pull-request that modifies the Mesh_3 plugin, and I will improve that dialog in the same PR.